uberenv.py

```diff
@@ -511,10 +535,13 @@ def patch(self):
             sexe("spack/bin/spack compiler find", echo=True)
 
         # hot-copy our packages into spack
-        if self.pkgs:
+        if self.spack_packages_base_paths:
             dest_spack_pkgs = pjoin(spack_dir,"var","spack","repos","builtin","packages")
-            print("[copying patched packages from {0}]".format(self.pkgs))
-            sexe("cp -Rf {} {}".format(self.pkgs,dest_spack_pkgs))
+            for _base_path in self.spack_packages_base_paths:
```

Comment:
The order in the dictionary is quite important, right? Maybe we should mention it in the doc?